silo-vaults/contracts/interfaces/IMetaMorpho.sol

```diff
@@ -198,7 +198,8 @@ interface IMetaMorphoStaticTyping is IMetaMorphoBase {
 
 /// @title IMetaMorpho
 /// @author Morpho Labs
-/// @custom:contact security@morpho.org
+/// @custom:modified Silo Labs
```

Comment:
Rather than modified tag, we should put Silo as author and add comment about forking from morpho with gratitude

Reply:
Done. Take a look.